Makefile

```diff
@@ -1,7 +1,13 @@
-.PHONY: test clippy docker-shell nix-shell benchmarks benchmark docs
+.PHONY: test clippy docker-shell nix-shell benchmarks benchmark docs py.develop
 
-test:
+py.develop:
+	python3 -m venv .venv
+	. .venv/bin/activate
+	maturin develop -m py/Cargo.toml
+
+test: py.develop
 	cargo test
+	python3 -m unittest py/tests/lamdaworks_py_test.py
```

Comment:
Invert the order of these. The first target is the default target in `make`, which would be expected to be either `test` or `build`.